storybook/stories/AdaRedemptionDialog.stories.js

```diff
@@ -28,21 +29,34 @@ storiesOf('AdaRedemptionForm', module)
       <AdaRedemptionForm
         onSubmit={action('submit')}
         isSubmitting={false}
-        isRedemptionDisclaimerAccepted={true}
+        isRedemptionDisclaimerAccepted
         isCertificateSelected={false}
         isCertificateEncrypted={false}
         onCertificateSelected={action('onCertificateSelected')}
         onPassPhraseChanged={action('onPassPhraseChanged')}
         onRedemptionCodeChanged={action('onRedemptionCodeChanged')}
         onChooseRedemptionType={action('onChooseRedemptionType')}
-        redemptionCode=''
-        redemptionType='regular'
+        redemptionCode=""
+        redemptionType="regular"
```

Comment:
@daniloprates please replace `regular` magic word with `ADA_REDEMPTION_TYPES.REGULAR` from this file: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/types/redemptionTypes.js#L5

Reply:
Done! ☝🏽